src/GameLogic/PlayerActions/Skills/SummonPartySkillPlugin.cs

```diff
@@ -0,0 +1,143 @@
+﻿// <copyright file="SummonPartySkillPlugin.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.GameLogic.PlayerActions.Skills;
+
+using System.Runtime.InteropServices;
+using System.Threading;
+using MUnique.OpenMU.GameLogic.Views;
+using MUnique.OpenMU.GameLogic.Views.World;
+using MUnique.OpenMU.Pathfinding;
+using MUnique.OpenMU.PlugIns;
+
+/// <summary>
+/// The summon skill action for Dark Lord.
+/// </summary>
+[PlugIn(nameof(SummonPartySkillPlugin), "Handles the summon party skill of the dark lord class.")]
+[Guid("44e34497-c9e1-4c15-9388-589dfa3fa820")]
+public class SummonPartySkillPlugin : TargetedSkillPluginBase
+{
+    private static readonly int CountdownSeconds = 5;
+
+    /// <inheritdoc />
+    public override short Key => 63;
+
+    /// <inheritdoc />
+    public override async ValueTask PerformSkillAsync(Player player, IAttackable target, ushort skillId)
+    {
+        var skillEntry = player.SkillList?.GetSkill(skillId);
+        if (skillEntry?.Skill is null)
+        {
+            return;
+        }
+
+        if (player.Party == null)
```

Comment:
Please check with `is null`

Reply:
Updated. Also, latest commit already has map requirement checking before warping the target member.